models/credmark/protocols/dexes/curve/curve_finance.py

```diff
@@ -2,23 +2,79 @@
 from credmark.cmf.model import Model
 from credmark.cmf.model.errors import ModelRunError
 from credmark.cmf.types.ledger import TransactionTable
-from credmark.dto import DTO
+
+from credmark.dto import (
+    DTO,
+    EmptyInput,
+)
+
 from credmark.cmf.types import (
     Address,
     Account,
     Accounts,
     Contract,
+    Contracts,
     Token,
-    Tokens
+    Tokens,
 )
-from credmark.cmf.types.series import BlockSeries
 
-from models.tmp_abi_lookup import (
-    CURVE_SWAP_ABI_1,
-    CURVE_SWAP_ABI_2,
-    CURVE_REGISTRY_ADDRESS,
-    CURVE_REGISTRY_ABI
-)
+
+@Model.describe(slug='curve-fi.get-provider',
+                version='1.1',
+                display_name='Curve Finance - Get Provider',
+                description='Get provider contract',
+                input=EmptyInput,
+                output=Contract)
+class CurveFinanceGetProvider(Model):
+    CURVE_PROVIDER_ALL_NETWORK = '0x0000000022D53366457F9d5E68Ec105046FC4383'
```

Comment:
This is cross chain?

Reply:
It exists where curve made the deployed. From Curve's doc:
> The address provider is deployed to the same address on Ethereum and all sidechains/L2’s where Curve is active.

I checked a few that's correct.
- https://snowtrace.io/address/0x0000000022D53366457F9d5E68Ec105046FC4383
- https://polygonscan.com/address/0x0000000022D53366457F9d5E68Ec105046FC4383